rosette/src/Ob.cc

```diff
@@ -1462,6 +1465,18 @@ DEF("cwd",sysCwd, 0, 0)
 	return PRIM_ERROR(buf);
 }
 
+// TODO: Swap with something crypotgraphically secure and uniform in length
+DEF("random-number-init", randomNumberInit, 0, 0)
```

Comment:
Is this used for anything other than generating fresh names?

Reply:
Nope; I was thinking I should move it to the "big bang" but I didn't really want to touch the code around there.